src/clj/puppetlabs/services/jruby/jruby_puppet_agents.clj

```diff
@@ -161,7 +161,8 @@
   (let [{:keys [config pool-state]} pool-context
         new-pool-state (jruby-internal/create-pool-from-config config)
         old-pool @pool-state]
-    (swap-and-drain-pool! pool-context old-pool new-pool-state true)))
+    (swap-and-drain-pool! pool-context old-pool new-pool-state true)
+    (println "*** done flushing and repopulating pool")))
```

Comment:
Nevermind, should have read the commit message before commenting on this. (facepalm)